docs/calendar-draft.md

```diff
@@ -352,8 +359,9 @@ Temporal.Date.from = function(thing: string | object, options: object) {
 As discussed earlier, Temporal.Date will defer to Temporal.Calendar methods wherever necessary.  Example implementation of selected Temporal.Date methods:
 
 ```javascript
-Temporal.Date.prototype.plus = function(duration) {
-	return this.calendar.plus(this, duration);
+Temporal.Date.prototype.plus = function(duration, options) {
+	const constructor = this.constructor[Symbol.species] || Temporal.Date;
```

Comment:
Note, this isn't *exactly* what SpeciesConstructor does, but it's close enough for now.

Reply:
Yeah, this was my attempt at doing "approximately SpeciesConstructor" in a pseudocode one-liner, I considered actually putting `ES.SpeciesConstructor(this, Temporal.Date)` but felt that would be more confusing :smile:

Reply:
If you have ES.SpeciesConstructor available, I'd use it.

Reply:
"available" is maybe not the right word here, as this code is for illustration and will never be run, but I've made this change.